apps/zui/src/js/models/program.test.ts

```diff
@@ -57,21 +58,22 @@ describe("drill down", () => {
     count: 24,
   })
 
-  async function run(value: any, text: string) {
-    const info = await fetchQueryInfo(text)
-    return drillDown(text, value, info.isSummarized, info.groupByKeys)
+  async function run(value: any, text: string, pool?: string) {
+    const d = await fetchQueryInfo(text, pool)
```

Comment:
Nits: Every call to this passes `"test"` as `pool` so no need for a parameter. And `d` is a surprising name for the result of `fetchQueryInfo()`.
```suggestion
  async function run(value: any, text: string) {
    const info = await fetchQueryInfo(text, "test")
```